### ISSUE TITLE
kubernetes controlplane component fail to re-install after deletion

### ISSUE THREAD

**Issue by knrt10** [kinvolk/lokomotive#1255]
**Description**

If we un-install kubernetes controlplane component via helm, on re-applying cluster it does not creates required resources and times out

**Impact**

In-availability of Kubernetes controlplane components

**Environment and steps to reproduce**

1. **Set-up**: Create a single node cluster
2. **Task**: Run `helm uninstall -n kube-system kubernetes`
3. After above step re-apply the cluster `lokoctl cluster apply -v`

**Expected behavior**
controlplane component kubernetes should re-install before timeout

**Comment by invidian** [comment]
Yeah, this is because we don't checkpoint kube-controller-manager and kube-scheduler (#1044).
